### PR TITLE
Uploaded a blog about quick deploy

### PR DESCRIPTION
**Pull request by soyeric128**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Uploaded a blog about quick deploy

## Changelog

- Not for changelog (changelog entry is not required)

## Related Issues

Fixes #issue

### CODE REVIEW

**Comment by soyeric128** [comment]
Please help review this blog @wubx ,thanks

File: website/blog/20220609-Quick-Deploy.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+---
+title: Deploying Databend on Your Laptop in Minutes
+description: This is my first post on Databend.
+slug: Quick-Deploy
```
Comment (Xuanwo) [inline website/blog/20220609-Quick-Deploy.md:4]: No need to set slug unless we have some special requirement.

File: website/blog/20220609-Quick-Deploy.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+c. Run the following command to start databend-meta:
+
+```shell
+./databend-query -c ../etc/databend-query.toml > query.log 2>&1 &
+```
+
+d. Run the following command to check if databend-meta was started successfully: 
+
+```vb
```
Comment (Xuanwo) [inline website/blog/20220609-Quick-Deploy.md:120]:
```suggestion
```shell
```

File: website/blog/20220609-Quick-Deploy.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+```shell
+curl -I  http://127.0.0.1:8101/v1/health
+```
+
+## STEP 3. Deploying a Standalone databend-query
+
+a. Create a file named `databend-meta.toml` in the folder `/usr/local/databend/etc` with the following content:
+
+```
```
Comment (Xuanwo) [inline website/blog/20220609-Quick-Deploy.md:67]:
```suggestion
```toml
```

File: website/blog/20220609-quick-deploy.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+  tags: [databend, deploy]
+---
+
+**Deploying Databend on Your Laptop in Minutes**
+
+
+Deploying a data warehouse sounds like a big job to you? Definitely NOT. Databend can be deployed to your laptop and uses the local file system as storage. You can complete the deployment in a few minutes even if you're new to Databend.
+Now let's get started!
+
+:::Note
```
Comment (Xuanwo) [inline website/blog/20220609-quick-deploy.md:17]:
```suggestion
:::tip
```

**Pull request merged**